PracticeRobot2/src/main/java/frc/robot/Robot.java

```diff
@@ -145,6 +146,10 @@ public void autonomousInit() {
     m_autoSelected = m_chooser.getSelected();
     // m_autoSelected = SmartDashboard.getString("Auto Selector", kDefaultAuto);
     System.out.println("Auto selected: " + m_autoSelected);
+    m_robotDrive.arcadeDrive(m_stick.getX(), m_stick.getY());
```

Comment:
Remember, this is in autonomousInit(), where we don't need to drive w/ the joystick. So you can remove this